### ISSUE TITLE
Ensure links are distinguished from surrounding text in a way that does not rely on color (#banner-link-updates):

### ISSUE THREAD

**Issue by aniketr99** [microsoft/vscode-docs#7276]
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".**

## GitHub Tags:
#A11yTCS; #A11ySev2; #A11yMAS; #DesktopWeb; #Chrome; #WCAG1.4.1; #WCP; #Use of Color; #Visual Studio Code Services; #A11yAuto; #AI4W; #BM_VisualStudioCodeServices_Web_Apr2023; #Win11;#clink-in-text-block;

## Environment Details: 
Application Name:  code.visualstudio.com
URL:https://code.visualstudio.com/docs/java/java-project
Microsoft Edge version 125.0.0.0 (Official Build) (64-bit)

## Repro Steps:
Hit the URL: https://code.visualstudio.com/docs/java/java-project
Run fast pass and observe the issue.

## Issue: 
Ensure links are distinguished from surrounding text in a way that does not rely on color (link-in-text-block - https://dequeuniversity.com/rules/axe/4.7/link-in-text-block?application=msftAI)

## Target application: 
Java project management in Visual Studio Code - https://code.visualstudio.com/docs/java/java-project

## Element path: 
#banner-link-updates

## Snippet: 
< a href="/updates" id="banner-link-updates">Version 1.89</a>

## Related paths: 
  .message

### Fix any of the following:
  The link has insufficient color contrast of 1.02:1 with the surrounding text. (Minimum contrast is 3:1, link text: #4daafc, surrounding text: #9ba3b4)
  The link has no styling (such as underline) to distinguish it from the surrounding text

## Environment: 
Microsoft Edge version 125.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
## Attachment:
![image](https://github.com/microsoft/vscode-docs/assets/100269580/bfa9d2c9-cee0-4f4d-be80-6830d2d4dd58)

**Comment by ntrogh** [comment]
@daviddossett Can you look into this accessibility issue for the version banner?

**Comment by daviddossett** [comment]
This was fixed in https://github.com/microsoft/vscode-website/pull/1753

![Image](https://github.com/microsoft/vscode-docs/assets/25163139/869be614-b9ba-4126-989b-e8c092c48046)

**Comment by rzhao271** [comment]
@aniketr99 could you test the page again? You can access it at https://code.visualstudio.com/docs/java/java-project?dark-plus-v2

**Comment by sbatten** [comment]
@aniketr99 this is fixed, can we please close this issue?

**Comment by rzhao271** [comment]
Closing due to lack of activity.
